examples/testprograms/http/http.star

```diff
@@ -3,11 +3,12 @@ load("@http", "myhttp")
 def foo(x):
     return "meow, " + x
 
-def on_http_get():
+def on_http_get(conn):
+    print(conn)
     print(time.now())
     print(store.set("x", "meow"))
 
-    resp, err = myhttp.get("https://httpbin.org/status/404", ak.callopts(catch=True))
+    resp, err = conn.get("https://httpbin.org/status/404", ak.callopts(catch=True))
```

Comment:
so you pass an actual reference to the connection and not just an id ? do we really want to do it ?
not sure starlark lsp could understand the types here vs getting connectionID and then selecting the correct loaded one